index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+  
+
+
+	<div class="header-background" >
+		<div class="nav">
+		<a href="" class="m-logo" >My Logo</a>
+
+		<ul>
+         <li class="link-right">
+              
+              <a href="/someurl" class="inline"><b><img src="images/menu.PNG" alt="ss"></b></a>
+          </li>
+         </ul>
+     </div>
+
+
+
+        <div class="section1">
+          
+        	<img src="images/l1.PNG" alt="abc" class="l1">
+        	<img src="images/m.PNG" alt="abc" class="m">
+          <img src="images/o.PNG" alt="abc" class="m">
+        	<img src="images/r.png"  alt="abc" class="image3 dkl" >
+
+        
+
+        </div>
+
+
+
+
+
+
+
+
+        <div class="section2">
+        	
+        	<h2 class="h2 imelex">I’m Aziz un Nabi 
+                        Glad to see you!</h2>
+                        <h4  class="sec3p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h4>
```

Comment:
- [ ] Kindly make sure you use a `p` tag as this is a paragraph text and not a heading. 🙏